### PR TITLE
Add new templates and static files

### PR DESCRIPTION
**Pull request by rodionvolovik**
- base.jinja
- person.jinja
- _connections.jinja
- _declarations.jinja

### CODE REVIEW

File: pepdb/core/templates/person.jinja [hunk 1]
```diff
@@ -164,336 +113,1386 @@
...
-                                            {{ render_related_persons(related_persons.personal, "personal") }}
-                                        </ul>
-                                    </li>
+                                    {% if person.reputation_crimes_uk %}
+                                        <h4>{{ _("Кримінальні провадження") }}</h4>
+                                        {{ person.reputation_crimes_uk|richtext }}
                                     {% endif %}
-                                    {% if related_persons.business %}
-                                    <li>
-
```
Comment (dchaplinsky) [inline pepdb/core/templates/person.jinja]: Keep that please

File: pepdb/core/templates/base.jinja [hunk 2]
```diff
@@ -1,11 +1,9 @@
 <!DOCTYPE html>
 <html lang="{{ LANGUAGE_CODE }}">
 <head>
-    <meta charset="utf-8">
+    <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="{% block page_description %}{% endblock %}">
-    <meta name="author" content="">
-    <meta name='yandex-verification' content='5773e8febf6e53a1' />
```
Comment (dchaplinsky) [inline pepdb/core/templates/base.jinja]: Keep that as well, please.

File: pepdb/core/templates/base.jinja [hunk 3]
```diff
@@ -24,175 +22,206 @@
...
+
+
+    <footer class="footer">
+        <div class="footer__top">
+            <div class="content">
+                <div class="footer__nav">
+                    <nav class="menu">
+                        <ul class="menu__list">
+                            <li class="menu__list-item">
+                                <a href="about-us.html">Про реєстр</a>
```
Comment (dchaplinsky) [inline pepdb/core/templates/base.jinja]: {{ _("Про реєстр") }}
here in below to keep i18n enabled.

File: pepdb/core/templates/person.jinja (hunk 1 above)
Comment (rodionvolovik) [reply pepdb/core/templates/person.jinja]: According to the design, there is only one section for all related persons. I use person.all_related_persons.all to visualize the whole list of relations. Is it necessary also to separately include related_persons.business anyway?

File: pepdb/core/templates/base.jinja (hunk 2 above)
Comment (rodionvolovik) [reply pepdb/core/templates/base.jinja]: Done

File: pepdb/core/templates/person.jinja (hunk 1 above)
Comment (dchaplinsky) [reply pepdb/core/templates/person.jinja]: Okay, noted.

File: pepdb/core/static/js/chart.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+// Set a callback to run when the Google Visualization API is loaded.
+google.charts.setOnLoadCallback(drawChart);
+
+// Callback that creates and populates a data table,
+// instantiates the pie chart, passes in the data and
+// draws it.
+function drawChart() {
+    // Create the data table.
+    var dataCashAssetsLineChart = google.visualization.arrayToDataTable([
+        ['Рік', 'Доходи декларанта', 'Доходи родини'],
```
Comment (dchaplinsky) [inline pepdb/core/static/js/chart.js]: We will need to negotiate how to provide you this data.

I can do ajax endpoint or serialize it into json into data attributes.

File: pepdb/core/templates/macros/_connections.jinja [hunk 5]
```diff
@@ -1,207 +1,130 @@
-{% macro render_connection(connection, id) %}
```
Comment (dchaplinsky) [inline pepdb/core/templates/macros/_connections.jinja]: Keep in mind, that this code is also used in company.jinja template (at least for now).

**Pull request merged**